public/pages/SignIn/SignIn.page.tsx

```diff
@@ -44,7 +44,7 @@ export default class SignInPage extends React.Component<any, any> {
         </div>
         <div className="text-center w-max-4xl mx-auto mb-4">{Fider.session.tenant.isPrivate ? <Private /> : <Locked />}</div>
 
-        <SignInControl onEmailSent={this.onEmailSent} useEmail={true} redirectTo={Fider.settings.baseURL} />
+        <SignInControl onEmailSent={this.onEmailSent} useEmail={Fider.session.tenant.isAllowingEmailAuth} redirectTo={Fider.settings.baseURL} />
```

Comment:
I'm not sure we can reuse the useEmail to be honest. If you look at signup.page.tsx, you'll notice that it has some problems there.

<img width="526" alt="Screenshot 2021-07-20 at 21 07 40" src="https://user-images.githubusercontent.com/94755/126388684-1ba63de6-6349-4e10-97aa-95def58bd573.png">

Reply:
Oh damn I didn't noticed this one :frowning: 
I'll introduce a new state, it's not a big deal
Rolled back uses of `useEmail` and added `showEmailForm` state